tools/mlir-proto-opt/CMakeLists.txt

```diff
@@ -4,14 +4,35 @@ if (SANDBOX_ENABLE_ALP)
   )
 endif()
 
+get_property(dialect_libs GLOBAL PROPERTY MLIR_DIALECT_LIBS)
+get_property(conversion_libs GLOBAL PROPERTY MLIR_CONVERSION_LIBS)
+
 add_llvm_executable(mlir-proto-opt
   mlir-proto-opt.cpp
 )
 
+set(LIBS
+  ${dialect_libs}
+  ${conversion_libs}
+  ${test_libs}
```

Comment:
Where is `test_libs` defined?

Reply:
Not defined and not needed until we start registering test passes.